grails-app/domain/mama/ng/scheduler/Schedule.groovy

```diff
@@ -0,0 +1,30 @@
+package mama.ng.scheduler
+
+class Schedule {
+
+    static hasMany = [messages: Message]
+
+    String id = UUID.randomUUID().toString()
+    String userId
```

Comment:
No, we discussed this before. User_id here is the actual user receiving the message, thus this value is received from the user calling the service. Perhaps it should be renamed for clarity.